modules/phase_field/src/userobjects/EulerAngleUpdater.C

```diff
@@ -158,6 +158,20 @@ EulerAngleUpdater::initialize()
   _first_time = false;
 }
 
+// This function is currently unused but is needed for getGlobalID call in EulerAngleProvider.h
```

Comment:
What @permcody said: make the base class method _not_ pure virtual, then you don't need to add these methods here and below.